.concourse/pr.yml

```diff
@@ -0,0 +1,70 @@
+---
+resource_types:
+- name: pull-request
+  type: docker-image
+  source:
+    repository: teliaoss/github-pr-resource
```

Comment:
is it possible to lock this down to a tag?

Reply:
Done!